web-client/src/presenter/sequences/openPrintableTrialSessionWorkingCopyModalSequence.js

```diff
@@ -0,0 +1,13 @@
+import { clearModalStateAction } from '../actions/clearModalStateAction';
+import { setShowModalFactoryAction } from '../actions/setShowModalFactoryAction';
+import { state } from 'cerebral';
+
+const setCaseNotesFlagDefaultAction = ({ store }) => {
+  store.set(state.modal['caseNotesFlag'], true);
+};
```

Comment:
can you split this action out into its own file and add a test please?

Reply:
Done!